packages/compass-connection-import-export/src/hooks/use-export-connections.ts

```diff
@@ -29,12 +30,12 @@ const INITIAL_STATE: Readonly<ExportConnectionsState> = Object.freeze({
 });
 
 function connectionInfosToConnectionShortInfos(
-  infos: Pick<ConnectionInfo, 'favorite' | 'id'>[],
+  infos: Pick<ConnectionInfo, 'favorite' | 'id' | 'connectionOptions'>[],
   existingShortInfoList?: ConnectionShortInfo[]
 ): ConnectionShortInfo[] {
   return infos.map((conn) => ({
     id: conn.id,
-    name: conn.favorite?.name ?? '',
+    name: getConnectionTitle(conn),
```

Comment:
Not a blocker, might be worth updating the test here to protect against a regression https://github.com/mongodb-js/compass/blob/08662c42308747fe9e0af5aafa95d9a4aeeb8668/packages/compass-connection-import-export/src/hooks/use-export-connections.spec.tsx#L248